controllers/repository_test.go

```diff
@@ -34,6 +35,20 @@ func TestCreateRepository(t *testing.T) {
 	}
 }
 
+func TestCreateRepositoryFailsWithSlashes(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockECR := mock.NewMockECRAPI(ctrl)
+	controller := NewRepositoryController(mockECR)
+
+	c := generateContext(t, models.CreateRepositoryRequest{Name: "slash/test"}, map[string]string{"owner": "user"})
+	_, err := controller.CreateRepository(c)
+	if !strings.ContainsAny(err.Error(), "cannot contain '/' characters") {
```

Comment:
Is there a better way to check that the correct error message is being generated than this string membership?